connection.js

```diff
@@ -27,20 +27,49 @@ const insert_default_data = async () => {
   }
 }
 
-const select_all_data = async () => {
+const select_all_data_name = async () => {
   let { data, error } = await supabase
-    .from('countries')
+    .from(table_name)
     .select('name')
 
   if (error) {
     console.error(error)
   }
   if (data) {
-    console.log(data)
+    return data
+  }
+}
+const select_all_data_id = async () => {
+  let { data, error } = await supabase
+    .from(table_name)
+    .select('id')
+
+  if (error) {
+    console.error(error)
+  }
+  if (data) {
+    return data
+  }
+}
+
+const delete_country = async (country_id) => {
+  const { error } = await supabase
+    .from(table_name)
+    .delete()
+    .eq('id', country_id)
+  if (error) {
+    console.error(error)
   }
 }
 const main = async () => {
-  await select_all_data()
+  await insert_default_data()
+  const names = await select_all_data_name()
+  names.forEach(name => console.log(name))
+  const ids = await select_all_data_id()
+  for (let i = 0; i < ids.length; i += 1) {
+    console.log(`deleting id ${ids[i].id}`)
+    await delete_country(ids[i].id)
+  }
 }
 
-main().then(console.log).catch(console.error)
+main().then().catch(console.error)
```

Comment:
What will happen at `.then()`, now that the `console.log()` is removed?

Reply:
Since `main()` was returning nothing from the function it was console logging `undefined`.
Now there is no more undefined printed on to the console.